package.json

```diff
@@ -8,7 +8,7 @@
     "Adam Rudd <damvrr@gmail.com>"
   ],
   "scripts": {
-    "test": "tape test.js | faucet",
+    "test": "tape test.js | faucet && standard",
     "ci": "tape test.js && node testRandom"
```

Comment:
Yes, we need to have `standard` run in Travis.
`tap-spec` works better on Windows.